skrifa/src/color/mod.rs

```diff
@@ -1,6 +1,453 @@
-//! Color outlines.
+//! Drawing color glyphs.
+//!
+//! # Examples
+//! ## Retrieve the clip box of a COLRv1 glyph if it has one:
+//!
+//! ```
+//! # use skrifa::{scale::*, instance::Size, color::{ColorGlyphFormat, ColorPainter}};
+//! # use read_fonts::GlyphId;
+//! # fn get_colr_bb(font: read_fonts::FontRef, color_painter_impl : &mut ColorPainter, glyph_id : GlyphId) {
+//! match font.color_glyphs()
+//!       .get_with_format(glyph_id, ColorGlyphFormat::ColrV1)?
+//!       .bounding_box(&[], size)
+//!       .ok()?
+//! {
+//! Some(bounding_box) => {
+//!     println!("Bounding box is {:?}", bounding_box);
+//! }
+//! None => {
+//!     println!("Glyph has no clip box.")
+//! }
+//! }
+//! # }
+//! ```
+//!
+//! ## Paint a COLRv1 glyph given a font, and a glyph id and a [`ColorPainter`] implementation:
+//! ```
+//! # use skrifa::{scale::*, instance::Size, color::{ColorGlyphFormat, ColorPainter}};
+//! # use read_fonts::GlyphId;
+//! # fn paint_colr(font: read_fonts::FontRef, color_painter_impl : &mut ColorPainter, glyph_id : GlyphId) -> Result<(), ColorError> {
+//! let color_glyph = font.color_glyphs().get_type(glyph_id, ColorGlyphFormat::ColrV1)?;
+//! color_glyph.paint(&[], color_painter_impl)?
+//! # }
+//! ```
+//!
+mod instance;
+mod transform;
+mod traversal;
 
-// Temporary until remaining API is filled in.
-#![allow(dead_code)]
+#[cfg(test)]
+mod traversal_tests;
 
-mod instance;
+use raw::tables::colr;
+#[cfg(test)]
+use serde::{Deserialize, Serialize};
+
+use read_fonts::{
+    tables::colr::{CompositeMode, Extend},
```

Comment:
One QOL improvement I missed: re-exporting `CompositeMode` and `Extend` here would be nice so users don’t have to dig through read-fonts for them.